packages/did/src/DidDetails/FullDidDetails.ts

```diff
@@ -130,7 +130,7 @@ export async function authorizeExtrinsic(
   }
 
   const [signingKey] = getKeysForExtrinsic(did, extrinsic)
-  if (!signingKey) {
+  if (signingKey === undefined) {
```

Comment:
Again no notion of empty arrays. I think we should rather avoid them.

Reply:
don't really understand what you mean

Reply:
we clarified it. TS is not complaining, that `signingKey` could be undefined.